projects/plugins/jetpack/extensions/blocks/blogroll/utils.js

```diff
@@ -2,8 +2,10 @@ import { createBlock } from '@wordpress/blocks';
 import PlaceholderSiteIcon from './placeholder-site-icon.svg';
 
 export function createBlockFromRecommendation( attrs ) {
+	const trimmedURL = attrs?.url?.replace( /^(?:https?:\/\/)?(?:www\.)?/i, '' ).split( '/' )[ 0 ];
```

Comment:
```suggestion
	const trimmedURL = new URL( attrs?.url? ).host.
```

Reply:
This will work for removing the protocol, but it won't remove 'www'.

Reply:
@alshakero - I'm going to combine the solutions to handle both cases.